src/main/java/org/apache/sysml/runtime/instructions/gpu/ConvolutionGPUInstruction.java

```diff
@@ -52,6 +53,15 @@ public ConvolutionGPUInstruction(CPOperand in1, CPOperand in2, CPOperand out, St
 		_output = out;
 	}
 	
```

Comment:
Sorry, saw ur comments after pushing the commit. I will add comments in a subsequent commit :)